apd/move_urls_from_descriptions.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/ruby -w
+
+require 'json'
+
+file_name = ARGV.first.to_s
+json = JSON.parse(File.open(file_name).read)
+
+json.each do |program|
+	program_location = program['locations'].first
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.